Sources/PT.PM.JavaScriptParseTreeUst.Tests/JavaScriptConverterTests.cs

```diff
@@ -11,10 +12,17 @@ public class JavaScriptConverterTests
         [Test]
         public void Convert_JavaScriptSyntaxFiles_WithoutErrors()
         {
-            Assert.Inconclusive("Waiting for the new version of https://github.com/sebastienros/esprima-dotnet");
+            var ignoredFiles = new []
+            {
+                "EnhancedRegularExpression.js",
+                "MapSetAndWeakMapWeakSet.js",
+                "Modules.js",
+                "Outdated.js"
+            };
 
             TestUtility.CheckProject(Path.Combine(TestUtility.GrammarsDirectory, "javascript", "examples"),
-                Language.JavaScript, Stage.Ust);
+                Language.JavaScript, Stage.Ust, searchPredicate: fileName =>
+                    ignoredFiles.All(ignoredFile => !fileName.Contains(ignoredFile)));
```

Comment:
Maybe `!fileName.EndsWith(ignoredFile)` will better convey the intention? And hopefully work little faster? :)